src/App.jsx

```diff
@@ -1,46 +1,22 @@
-import { useState, useEffect } from "react";
+import { useState } from "react";
 import "./App.css";
 // import WrapTheList from "./WrapTheList";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 
 function App() {
-  // const [ todoList, setTodoList ] = useState([]);
-  const [ newTodo, setNewTodo ] = useState('');
-
-  const todoList = [
-    {
-      id: 0,
-      title: 'Read 37 pages of "The Road To React 2023',
-    },
-    {
-      id: 1,
-      title: "Watch videos from week 1",
-    },
-    {
-      id: 2,
-      title: "Complete week 1 assignment",
-    },
-  ];
-
-  useEffect(()=>{
-    console.log("useEffect!!! NewTOdo:", newTodo);
-
-  }, [newTodo])
-
-
+  const [ todoList, setTodoList ] = useState([]);
 
+  function addTodo(newTodo){
+    setTodoList([...todoList, newTodo]);
+  }
 
   return (
     <>
       <h1>Todo List</h1>
       <TodoList content={todoList}></TodoList>
-      <AddTodoForm onAddTodo={(text) => {setNewTodo(text)}}/>
-      <p>{ newTodo }</p>
-
-      {/* This is a component I imagined and decided to implement.
-      <WrapTheList list={todoList} mainTagName = 'ul'/> */}
+      <AddTodoForm onAddTodo={(item) => {addTodo(item)}}/>
```

Comment:
I believe you could make the value for `onAddTodo` here "`{addTodo}`" without the surrounding function. Either way is probably fine but this might help make your code a bit more compact and readable. Give it a try if you're curious!

Reply:
You are right, it works! But how does the`addTodo` function know the argument we use? I thought we have to use callbacks for functions with arguments...